src/js/modules/articles/components/ArticlesList.js

```diff
@@ -4,40 +4,38 @@
 import React, { Component } from "react";
 import Article from "./Article";
 import injectSheet from "react-jss";
-import { connect } from 'react-redux';
+import { connect } from "react-redux";
 
 const styles = {
   articlesList: {
-    listStyle: "none",
+    listStyle: "none"
   },
   articlesListContainer: {
     padding: "5%"
   }
 };
 
-const ArticlesList = ({classes, articles}) => (
+const ArticlesList = ({ classes, articles }) =>
   <div className={classes.articlesListContainer}>
     <h2> Articles </h2>
     <ul className={classes.articlesList}>
-    {articles.map((article, index) =>
-        <li key={`articleLI${index}`}>
+      {articles.map(article =>
+        <li key={`articleLI${article.id}`}>
           <Article
```

Comment:
consider new lines inside the braces in this statement (spans from line 22 to 28)